### PR TITLE
drinksbutkateisntanidiot

### PR DESCRIPTION
**Pull request by katebishop49**
if this doesnt work im shooting miself

#i managed to make sodas pull out fire#

#I have no idea, anymore this was a terrible idea#

## Changelog
:cl:
add:soda now pulls out fires, soda cans can be altclicked to pull the safety of it
/:cl:

### CODE REVIEW

File: code/modules/food_and_drinks/drinks/drinks.dm [hunk 1]
```diff
@@ -488,6 +496,114 @@
 	if(!is_drainable())
 		open_soda(user)
 	return ..()
+/obj/item/reagent_containers/food/drinks/soda_cans/AltClick(mob/user)
+	if(spillable == TRUE)
+		to_chat(user, "\the [src] doesnt have enougth preasure.")
```
Comment (TriggeredBoi) [inline code/modules/food_and_drinks/drinks/drinks.dm:501]:
```suggestion
		to_chat(user, "\the [src] doesnt have enougth pleasure.")
```

Comment (katebishop49) [reply code/modules/food_and_drinks/drinks/drinks.dm:501]: begone horny

**Comment by katebishop49** [comment]
merge?

**Comment by LastCrusader105** [comment]
You have to alt click to drink it as well?
Or that remain unchanged?

**Pull request merged**